pkg/ddevapp/global_dotddev_assets/commands/host/self-upgrade

```diff
@@ -7,6 +7,8 @@
 ## Example: "ddev self-upgrade"
 ## CanRunGlobally: true
 
+# split the loop output below by newlines
+IFS=$'\n'
 for ddev_path in $(which -a ddev); do
```

Comment:
How about this to avoid the situation where there are multiple repetitions of something in the path?

```suggestion
for ddev_path in $(which -a ddev | uniq); do
```

Reply:
I believe the idea in

- #5834

was to show all possible locations so that people can see that they have DDEV in multiple locations.

I agree with `| uniq`, this is a nice addition.